CRITICAL - 2023-01-09 18:58:03 --> Undefined array key "criadopor"
#0 /Users/arseniomuanda/Arpetic/GitHub/meuruca/app/Helpers/funcao_helper.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', '/Users/arseniom...', 34)
#1 /Users/arseniomuanda/Arpetic/GitHub/meuruca/app/Controllers/Login.php(265): cadastronormal(Object(App\Models\ContaModel), Array, Object(CodeIgniter\Database\MySQLi\Connection), Object(App\Models\AuditoriaModel))
#2 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(935): App\Controllers\Login->newaccount()
#3 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#4 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Users/arseniomuanda/Arpetic/GitHub/meuruca/public/index.php(44): CodeIgniter\CodeIgniter->run()
#6 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(45): require_once('/Users/arseniom...')
#7 {main}
ERROR - 2023-01-09 19:13:07 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 1 in /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:314
Stack trace:
#0 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(314): mysqli->query('UPDATE `utiliza...')
#1 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/BaseConnection.php(715): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `utiliza...')
#2 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/BaseConnection.php(643): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `utiliza...')
#3 /Users/arseniomuanda/Arpetic/GitHub/meuruca/app/Controllers/Login.php(185): CodeIgniter\Database\BaseConnection->query('UPDATE `utiliza...')
#4 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(935): App\Controllers\Login->logout()
#5 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Users/arseniomuanda/Arpetic/GitHub/meuruca/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(45): require_once('/Users/arseniom...')
#9 {main}
CRITICAL - 2023-01-09 19:13:07 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 1
#0 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(314): mysqli->query('UPDATE `utiliza...')
#1 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/BaseConnection.php(715): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `utiliza...')
#2 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/BaseConnection.php(643): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `utiliza...')
#3 /Users/arseniomuanda/Arpetic/GitHub/meuruca/app/Controllers/Login.php(185): CodeIgniter\Database\BaseConnection->query('UPDATE `utiliza...')
#4 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(935): App\Controllers\Login->logout()
#5 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Users/arseniomuanda/Arpetic/GitHub/meuruca/public/index.php(44): CodeIgniter\CodeIgniter->run()
#8 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(45): require_once('/Users/arseniom...')
#9 {main}
ERROR - 2023-01-09 19:34:03 --> mysqli_sql_exception: Duplicate entry '382738232' for key 'matricula' in /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:314
Stack trace:
#0 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(314): mysqli->query('INSERT INTO `vi...')
#1 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/BaseConnection.php(715): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `vi...')
#2 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/BaseConnection.php(643): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `vi...')
#3 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(2265): CodeIgniter\Database\BaseConnection->query('INSERT INTO `vi...', Array, false)
#4 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/BaseModel.php(758): CodeIgniter\Model->doInsert(Array)
#6 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Model.php(705): CodeIgniter\BaseModel->insert(Array, false)
#7 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/BaseModel.php(667): CodeIgniter\Model->insert(Array, false)
#8 /Users/arseniomuanda/Arpetic/GitHub/meuruca/app/Helpers/funcao_helper.php(60): CodeIgniter\BaseModel->save(Array)
#9 /Users/arseniomuanda/Arpetic/GitHub/meuruca/app/Controllers/Operations.php(182): cadastrocomumafoto(Object(App\Models\ViaturaModel), Array, Object(CodeIgniter\Database\MySQLi\Connection), Object(App\Models\AuditoriaModel), NULL, 'imagem')
#10 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(935): App\Controllers\Operations->index()
#11 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(431): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operations))
#12 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/CodeIgniter.php(332): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /Users/arseniomuanda/Arpetic/GitHub/meuruca/public/index.php(44): CodeIgniter\CodeIgniter->run()
#14 /Users/arseniomuanda/Arpetic/GitHub/meuruca/vendor/codeigniter4/framework/system/Commands/Server/rewrite.php(45): require_once('/Users/arseniom...')
#15 {main}
